acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/ExecutorPluginPage.java

```diff
@@ -31,7 +31,7 @@ public ExecutorPluginPage(WebDriver driver) {
     public WebDriver driver;
 
     public void open() {
-        driver.get(base + "/blue/organizations/jenkins/executor-info");
```

Comment:
OMG how did I miss this. I expected it to be looking for the button and click it. :-/

Reply:
I missed it too :(